### PR TITLE
feat(platforms): init

### PR DESCRIPTION
**Pull request by divyenduz**
- [x] Netlify via CLI
- [x] Heroku

### CODE REVIEW

File: platforms/heroku/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    },
+  })
+
+  const users = await photon.users.findOne({
+    where: {
+      id: createUser.id,
+    },
+  })
+
+  const deleteManyUsers = await photon.users.deleteMany()
```
Comment (steebchen) [inline platforms/heroku/index.js]: afaik this should be `.deleteMany({})`

File: platforms/heroku/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  return res.send(
+    JSON.stringify({
```
Comment (steebchen) [inline platforms/heroku/index.js]: pretty sure you can just do `res.json({ ... }` without `JSON.stringify` :)

File: platforms/heroku/index.js (hunk 1 above)
Comment (divyenduz) [reply platforms/heroku/index.js]: types give a hiccup at times, but this is "js" and runtime is happy with this.

File: platforms/heroku/index.js (hunk 2 above)
Comment (divyenduz) [reply platforms/heroku/index.js]: Not super sure, will give it a go tomorrow

**Comment by steebchen** [comment]
just FYI, the frameworks error should disappear after upgrading prisma2 (so it'll be fixed automatically when you merge master into this branch or just merge into master)

**Pull request merged**